web3/deploy_contract.js

```diff
@@ -49,14 +49,34 @@ async function interact(newContractInstance, account, name) {
     });
 }
 
+// This will get the block by block number and display it.  
+async function getBlockByNumber(blockNum) {
+    console.log("\n", "block number: " + blockNum);
+    return web3.eth.getBlock(parseInt(blockNum))
+    .then(console.log); 
+}
+
 async function run() {
   const account = await getCurrentAccount();
+
   const contract = await deployContract(account);
   // TODO: Remove await from first interact when mempool nonce resolved
-  await interact(contract, account, true);
-  const blockNum = await interact(contract, account, false);
-  console.log("block number: " + blockNum);
-  // web3.eth.getBlock(parseInt(blockNum)).then(console.log);
+  
+  const blockNumT = await interact(contract, account, true);
+  console.log("block number: " + blockNumT, "\n");
+  
+  const blockNumF = await interact(contract, account, false);
+  console.log("block number: " + blockNumF, "\n");
```

Comment:
So one thing that is undocumented (because this was really only used by me for testing) is that the await was added on the first transaction when the pending tx nonce wasn't being updated correctly (part 1 of https://github.com/ChainSafe/ethermint/issues/164) and can be updated back to not awaiting on the first transaction to be processed before sending the second if nonce is set manually.

The goal for this was to test that the block bloom filter would include the bitwise OR of both transaction bloom filters, which was being tested by skipping certain checks and removing the first await, but the first await was just added to allow this to be functional without any other changes.

Out of scope of this PR, just want to give context on why the structure may have seemed off or you may get unexpected results. I just never updated this because I didn't have to

Reply:
I think it's ok to leave it this way for now.